src/Storages/MergeTree/MergeTreePartition.h

```diff
@@ -50,6 +50,7 @@ struct MergeTreePartition
     void assign(const MergeTreePartition & other) { value = other.value; }
 
     void create(const StorageMetadataPtr & metadata_snapshot, Block block, size_t row, ContextPtr context);
+    void createAndValidateMinMaxPartitionIds(const StorageMetadataPtr & metadata_snapshot, Block block, ContextPtr context);
```

Comment:
`block` does not says what it is, maybe better to call it `partition_by_min_max` ?